src/components/RadioButton/RadioButton.st.css

```diff
@@ -162,6 +178,19 @@
     box-sizing: border-box;
 }
 
+.root:box:disabled .wrapper {
+    /* overriding the styling from wix-ui-core */
+    filter: none;
+    opacity: 1;
+    /*******/
```

Comment:
I wonder if we want to provide a way to disable these properties in ui-core instead of overriding them here? 🤔

Reply:
Actually, I think that these css properties should be removed from the core and everyone should use its own design but it's breaking changes.
I think that it doesn't worth the effort, I already waste too much time on it.